translations/strings.yaml

```diff
@@ -63,3 +63,21 @@ settingsDev:
   useStagingTitle: Use Staging Servers
   useStagingSubtitle: Staging services are used to test features before they go live to production
   restartRequired: VPN must be restarted to take effect
+
+splittunnel:
+  missingAppActionButton:
+    value:
+      - Add apps manually
+    comment:
+      - Displayed in notification as "call to action" when apps are missing from the split-tunnel settings.
+  missingAppMultiple: 
+    value:
+      - \%0 apps missing from list.
+    comment:
+      - apps in the split-tunnel settings. %0 is the amount of apps missing
+  missingAppAddedOne: 
+    value:
+      - Success! You added the %0 to this list.
```

Comment:
This sounds off to me: `Success! You added the Instagram to this list.`

Either we have app after the name, or drop the "the". I think it would make sense to also have the name between quotes, i.e. `Success! You added “%0” to this list.` (imagine the app is called "Send to"…).